wordpress-mobile/WordPress-Android
# Create date selector bar in stats

Fixes #9137, #9067
This PR adds a date selecting bar to the DWMY stats. This bar is attached to the toolbar and doesn't hide on scroll. The enabled dates that could be iterated through the block are the same ones as in the Overview block. 
As part of this PR I'm also replacing the selected date in the title of the overview block with a bigger value for the selected date/unit (for example if there are selected Views and the value for the selected date is 10, there will be `10 Views` in the title).
I've removed the `Date` block since it no longer makes sense.

To test:
* Go to Stats/Insights
* There is no Date selector visible
* Go to Stats/DWMY
* The date selector appears
* The title of the Overview block is now the selected value
* Click on the buttons in the Date selector
  * The Selected date changes and new data is loaded
  * The right limit value disables the next button
  * The left limit value disables the previous button

![ripple_effect](https://user-images.githubusercontent.com/1079756/52200943-563d1980-286a-11e9-80d1-bd742a378c51.gif)

Update release notes:
